src/actor/ActorTemplate.js

```diff
@@ -330,17 +408,36 @@ exports = function(inherits) {
       });
     };
 
+    /** Shortcut to this.view.style.opacity
+        @var {number} ActorTemplate#opacity */
     this._addStyleProperty('opacity');
+    /** Shortcut to this.view.style.fliipX
+        @var {number} ActorTemplate#fliipX */
     this._addStyleProperty('flipX');
+    /** Shortcut to this.view.style.flipY
+        @var {number} ActorTemplate#flipY */
     this._addStyleProperty('flipY');
+    /** Shortcut to this.view.style.r
+        @var {number} ActorTemplate#rotation */
```

Comment:
Can we have `actor.r` as an alias for `rotation`? It gets pretty confusing if you're used to using `r`on `style`, and if you're having to switch between the two. Is it bad practice to have two names for the same thing?